RSignatureVerifier/Environment.swift

```diff
@@ -0,0 +1,68 @@
+internal protocol EnvironmentSetupProtocol {
+    var valueNotFound: String { get }
+    func value(for key: String) -> String?
+    func deviceModel() -> String
+    func osVersion() -> String
+    func sdkName() -> String
+    func sdkVersion() -> String
+    func languageCode() -> String?
+    func countryCode() -> String?
+}
+
+internal class Environment {
```

Comment:
can it be a struct instead of a class?

Reply:
`Bundle(for: )` complains that the parameter is not a class :/